src/applications/vaos/new-appointment/components/VAFacilityPage/FacilitiesNotShown.jsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
+import ExpandingGroup from '@department-of-veterans-affairs/formation-react/ExpandingGroup';
+import recordEvent from 'platform/monitoring/record-event';
+import FacilityPhone from '../../../components/FacilityPhone';
+import { GA_PREFIX } from '../../../utils/constants';
+
+const UNSUPPORTED_FACILITY_RANGE = 100;
+
+export default function FacilitiesNotShown({ facilities, sortMethod }) {
+  const [isOpen, setIsOpen] = useState(false);
+  useEffect(
+    () => {
+      setIsOpen(false);
+    },
+    [sortMethod],
+  );
+  useEffect(
+    () => {
+      if (isOpen) {
+        recordEvent({
+          event: `${GA_PREFIX}-facilities-not-listed-click`,
+        });
+      }
+    },
+    [isOpen],
+  );
+
+  const nearbyUnsupportedFacilities = facilities.filter(
+    facility =>
+      !facility.legacyVAR.directSchedulingSupported &&
+      !facility.legacyVAR.requestSupported &&
+      facility.legacyVAR[sortMethod] < UNSUPPORTED_FACILITY_RANGE,
+  );
+
+  if (!nearbyUnsupportedFacilities.length) {
+    return null;
+  }
+
+  const buttonClass = classNames(
+    'additional-info-button',
```

Comment:
The code below is copied and modified from `AdditionalInfo`. I'm not using that component directly because I need to programmatically collapse it, but `AdditionalInfo` uses local component state for that.

cc @cvalarida @bkjohnson, not sure if this situation is something you want to take note of.

Reply:
It's good to know that there is a use case for that. So hypothetically, would a component that exposes a public method be useful? i.e.:

```js

// Could be identified by a ref or some other way
const myComponent = document.querySelector('#my-additional-info-component')

myComponent.collapse()
```

Reply:
I think the "React-y" way of dealing with it would be to have an `open` prop and switch between using that or using component state. That'd be what I'd expect as a consumer of React components.